src/Pages/Chart/Components/Card.js

```diff
@@ -0,0 +1,167 @@
+import React, { useState } from "react";
+import * as config from "../../../config";
+import styled from "styled-components";
+
+const Card = props => {
```

Comment:
```suggestion
const Card = ({ handleSingleCheck, handleModal, checkedItems, rank, image, title, album, artist }) => {
```
- props 객체 미리 구조 분해 할당 적용해서 사용하실 수 있습니다!

Reply:
적용완료했습니다!